src/components/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slug } = useParams();
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {people.find(p => p.name === person.motherName) ? (
+          <Link
+            to={`/people/${people.find(p => p.name === person.motherName)?.slug}`}
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so it make sense to move it to the variable
check other places